bootstrap/sql/mysql/v005__drop_test_case_fk_from_topology_test_history.sql

```diff
@@ -0,0 +1,16 @@
+-- Copyright 2017 Hortonworks.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--    http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+ALTER TABLE `topology_test_run_histories` DROP FOREIGN KEY `fk_topology_test_run_histories_topology_test_run_case`, DROP KEY `fk_topology_test_run_histories_topology_test_run_case`;
```

Comment:
Do we need this if we dont add the FK constraint it in the above script?

Reply:
Yes I broke down file because of explicitly showing intention (like breaking down commits) but we could address it to v004 and leave comment explaining it. I'll address it.